### PR TITLE
fix(Switches): organise owners

### PR DESCRIPTION
**Pull request by mxdvl**
## What does this change?

Remove people that have left the organisation.
Use groups over individuals where relevant.

Is the owners field relevant, if it most of the time identical to the Switch Group?

Should we have automated checks for github accounts that are no longer inside @guardian?

## Does this change need to be reproduced in dotcom-rendering ?

- [X] No
- [ ] Yes (please indicate your plans for DCR Implementation)

### Tested

- [ ] Locally
- [ ] On CODE (optional)

### CODE REVIEW

**Review by jamesgorrie — COMMENTED** [review]
Maybe we can hoist the `sellByDate` to an `Experiment`? This way we can still have experiments expire - and [email people when they do](https://github.com/guardian/frontend/blob/main/admin/app/jobs/ExpiringSwitchesEmailJob.scala#L18), but not need an expiry on switches, which are long lasting. 

With `owners` - it seems like we're using `SwitchGroup` as a proxy for what team owns a features, but maybe this should be more explicit there?

File: common/app/conf/switches/PerformanceSwitches.scala [hunk 1]
```diff
@@ -183,7 +183,7 @@ trait PerformanceSwitches {
     "If switched on, explicitly request lazy loading of images on supporting browsers.",
     owners = Seq(Owner.withGithub("nicl")),
     safeState = On,
-    sellByDate = LocalDate.of(2029, 9, 3),
+    sellByDate = never,
```
Comment (mxdvl) [inline common/app/conf/switches/PerformanceSwitches.scala:186]: Introduced in https://github.com/guardian/frontend/pull/21584/files#diff-d05c492cbdd04e6bd2f8c1323c2e775bf74ce0b8675fd9dbb39ba9d8847f373eR196, it does not seem like this was supposed to ever expire.

File: common/app/experiments/Experiments.scala [hunk 2]
```diff
@@ -30,7 +30,7 @@ object DCRFronts
     extends Experiment(
       name = "dcr-fronts",
       description = "Use DCR for fronts",
-      owners = Seq(Owner.withGithub("dotcom")),
+      owners = Seq(Owner.withGithub("@guardian/dotcom-platform")),
```
Comment (mxdvl) [inline common/app/experiments/Experiments.scala:33]: We use a variety of:
- `SwitchGroup` names
- team emails in `*@guardian.co.uk` and `*@theguardian.com`
- Github team handles

I’m not entirely sure of the benefits of having so many options.

**Comment by mxdvl** [comment]
> Maybe we can hoist the `sellByDate` to an `Experiment`?

I think this can be addressed into another PR. I’m not entirely sure how hoisting works in Scala, so let’s pair on it?

> This way we can still have experiments expire - and [email people when they do](https://github.com/guardian/frontend/blob/main/admin/app/jobs/ExpiringSwitchesEmailJob.scala#L18), but not need an expiry on switches, which are long lasting.

Worth noting that currently, we will only email individuals or teams that have used the `Owner.withEmail` feature. Ones registered with `SwitchGroup` or Github will not receive anything. The web engineers and Dotcom platform should still get reminders.

> With `owners` - it seems like we're using `SwitchGroup` as a proxy for what team owns a features, but maybe this should be more explicit there?

Yes, I’m not sure what the benefit of this is if this is identical to the Switch’s group…

**Review by ioannakok — APPROVED** [review]
LGTM!

**Pull request merged**